packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.vitest.tsx

```diff
@@ -346,10 +347,15 @@ describe('MarkdownSynapse tests', () => {
       expect(container).toMatchSnapshot()
     })
     it('works with two inline widgets', () => {
-      const { container } = renderComponent({
+      const component = getComponent({
         markdown:
           '${buttonlink?text=sometext&url=#/Help/How%20It%20Works&highlight=true}${buttonlink?text=APPLY&url=#/Apply&highlight=true} ',
       })
+
+      const { container } = render(<MemoryRouter>{component}</MemoryRouter>, {
+        wrapper: createWrapper(),
+      })
```

Comment:
Can you keep the current test as is, and add a new one with a Router? This new test will make sense in the context of the other comment.